src/main/java/EtfRecommendService/webSocket/WebSocketBroadcaster.java

```diff
@@ -0,0 +1,70 @@
+package EtfRecommendService.webSocket;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.TextMessage;
+import org.springframework.web.socket.WebSocketSession;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+//세션 + 브로드캐스팅
+//전체 종목 시세는 모든 접속자에게 동일한 데이터 전송
+@Component
+public class WebSocketBroadcaster {
+    private final Set<WebSocketSession> sessions = ConcurrentHashMap.newKeySet();
+    private final Map<WebSocketSession, Set<String>> userSubscriptions = new ConcurrentHashMap<>();
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    //전체 목록 보기
+    public void register(WebSocketSession session){
+        sessions.add(session);
+    }
+
+    public void unregister(WebSocketSession session){
+        sessions.remove(session);
+    }
+
+    //전체 목록
+    public void broadcast(StockPriceData data){
+        try{
+            String json = objectMapper.writeValueAsString(data);
+            TextMessage message = new TextMessage(json);
+            for (WebSocketSession session:sessions){
+                if (session.isOpen()){
+                    session.sendMessage(message);
+                }
+            }
+        } catch (IOException e){
+            e.printStackTrace();
+        }
```

Comment:
예외를 단순히 출력만 하고 있는데, 이러면 운영 환경에서 문제를 추적하기 어려워져요.
```suggestion
        } catch (IOException e){
            log.error("메시지 브로드캐스팅 중 오류 발생", e);
            // 필요시 상위 레이어로 예외 전파 또는 복구 로직 구현
        }
```